lib/config/presets/internal/group.ts

```diff
@@ -583,6 +583,17 @@ const staticGroups = {
       },
     ],
   },
+  reactAdmin: {
+    description: 'Group react-admin packages together',
+    packageRules: [
+      {
+        matchPackageNames: ['react-admin'],
+        matchPackagePatterns: ['^ra-'],
+        groupName: 'react-admin packages',
+        groupSlug: 'react-admin',
+      },
+    ],
+  },
```

Comment:
```suggestion
```

forgot to revert this?

Reply:
Oooh, now I got how this works. Sorry, misunderstood it..
Adapted and re-tested, still working as expected